core/src/main/java/nl/nn/adapterframework/webcontrol/api/TransactionalStorage.java

```diff
@@ -386,6 +422,13 @@ public Response downloadPipeMessage(
 			throw new ApiException("Pipe ["+pipeName+"] not found!");
 		}
 
+		try {
+			// messageId is double URLEncoded, because it can contain '/' in ExchangeMailListener
+			messageId = URLDecoder.decode(messageId,"UTF-8");
+		} catch (UnsupportedEncodingException e) {
+			log.warn(e);
+		}
+
```

Comment:
Ik zie dit nu 6x verschijnen..

Reply:
Dat komt ook omdat er 6 keer dezelfde method in deze class zit